content/_index.md

```diff
@@ -7,10 +7,42 @@ BookToC: false
 
 # The Linux Userspace API (UAPI) Group
 
-The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, and run modern Linux operating systems.
+The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, run, and securely update modern Linux operating systems.
 It serves as a central gathering place for specs, documentation, and ideas.
 
-Follow [@uapi-group](https://twitter.com/uapi_group) on twitter to catch the latest news!
+The group is in the process of constituting itself and is exploring the problem space.
+In the general scope of the group are trusted boot mechanisms, kernel / disk image formats, TPM usage, safe and secure updates, and related topics.
+The group also cares about interop of image-based and/or immutable Linux distributions.
+The group is in its inception, so the list of topics is by no means complete.
+Other relevant topics related to the above might be added in the future.
+
+The UAPI group aims to establish specifications for common concepts shared among relevant parties.
+We are open to anyone who has an interest in constructive collaboration to join the discussions, and to represent the directions and interests of their projects and distributions.
+
+Contributing members include people from 
+Red Hat / Fedora CoreOS ( [@travier](https://github.com/travier), [@dvdhrm](https://github.com/dvdhrm), [@gicmo](https://github.com/gicmo)),
+Flatpak ([@wjt](https://github.com/wjt)), mkosi ([@behrmann](https://github.com/behrmann)),
+gnomeos ([@alatiera](https://github.com/alatiera)),
+Meta ([DaanDeMeyer](https://github.com/DaanDeMeyer)),
+Arch Linux ([@Foxboron](https://github.com/Foxboron)),
+systemd ([@bluca](https://github.com/bluca), [poettering](https://github.com/poettering), [@brauner](https://github.com/brauner)),
+Flatcar ([@jepio](https://github.com/jepio), [@pothos](https://github.com/pothos), [@t-lo](https://github.com/t-lo))
+and others.
```

Comment:
Why put names on the page? You can mention companies but there's no reason to link to individual contributors? And if so you should ask there permissions before doing so, I would think.

Reply:
I'm not even sure if you can say Red Hat for example. It's not that we are an official entity with ties to Red Hat or any other company. We're not paid by anyone to do this. I'm not sure this is a good idea.

Anyone else a thought on this?

Reply:
Makes sense, will update.